microprofile-fault-tolerance/README.adoc

```diff
@@ -0,0 +1,433 @@
+////
+This guide is _ported_ from the main Quarkus repository
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+include::../shared-doc/attributes.adoc[]
+
+= microprofile-fault-tolerance: MicroProfile Fault Tolerance
+:author: Tomas Hofman, Radoslav Husar
+:level: Intermediate
+:technologies: MicroProfile, Fault Tolerance
+
+[abstract]
+The `microprofile-fault-tolerance` quickstart demonstrates how to use Eclipse MicroProfile Fault Tolerance in {productName}.
+
+One of the challenges brought by the distributed nature of microservices is that communication with external systems is
+inherently unreliable. This increases demand on resiliency of applications. To simplify making more resilient
+applications, {productName} contains an implementation of the MicroProfile Fault Tolerance specification.
+
+In this guide, we demonstrate usage of MicroProfile Fault Tolerance annotations such as `@Timeout`, `@Fallback`,
+`@Retry` and `@CircuitBreaker`. The specification also introduces `@Bulkhead` and `@Asynchronous` interceptor bindings not
+covered in this guide.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+
+== The Scenario
+
+The application built in this guide simulates a simple backend for a gourmet coffee e-shop. It implements a REST
+endpoint providing information about coffee samples we have on store.
+
+Let's imagine, although it's not implemented as such, that some of the methods in our endpoint require communication
+to external services like a database or an external microservice, which introduces a factor of unreliability.
+
+== Solution
+
```

Comment:
It should be mentioned here that fault tolerance subsystem is disabled by default and how to enable it.

(I know that you prepared PR to make it default but I am not sure when it's going to be merged so I mention this in case this quickstart is done sooner)

Reply:
Done. Added steps.